shopping-list-web/src/CollapsibleComponents/CollapsibleCategory/CollapsibleCategory.tsx

```diff
@@ -22,7 +22,7 @@ export default function CollapsibleCategory(props: {
         ref={contentRef}
         style={open ? { height: childHeight + "px" } : { height: "0px" }}
       >
-        <ListDisplay items={props.items} />
+        <ListDisplay items={props.items} checked />
```

Comment:
😮 this is cool, I didn't know you could just write `checked` rather than `checked={true}`

Reply:
hehe, yeah, is snazzy